discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -86,6 +86,13 @@ def _update_day2_config(self, api_client: InventoryClient, cluster_id: str):
         )
 
     def _create(self) -> str:
+        olm_operators = [{"name": name} for name in self._config.olm_operators]
```

Comment:
Why in a new line and not set it directly?

Reply:
please elaborate

Reply:
You just moved that line to new line without using it elsewhere. I didn't understand that :)

Reply:
@osherdp was there a specific reason for this change?

Reply:
writing better code was the reason

Reply:
:stuck_out_tongue_winking_eye:

Reply:
reverted to the original state.